.github/workflows/ubuntu_python_matrix_test.yml

```diff
@@ -28,7 +28,8 @@ jobs:
         echo "Install extras..."
         pip install numpy scipy ipython openpyxl sympy pyodbc pyyaml networkx xlrd matplotlib dill
         pip install pandas # Pandas needs to be installed after its dependencies to work correctly for Python 2.7 
-        pip install seaborn pymysql pyro4 pint pathos 
+        pip install seaborn pymysql pyro4 pint pathos
+        pip install cplex
```

Comment:
Is there a reason to make this a separate `pip install` command?

Reply:
(answering my own question): yes.  This needs to be it's own command, as cplex is not available for all versions of python (notably not 3.8), so we need to update this so that failure to install cplex does not necessarily kill the job.  Something like this might work:
```
pip install cplex || python -c "import sys; assert sys.version_info[:2] in ((3,4), (3,8))"
```

Reply:
That said, it may not be that useful to perform the version check.  The following is probably sufficient:
```
pip install cplex || echo "CPLEX Community Edition is not available for ${{ matrix.python-version }}"
```

We should also propagate this to the Windows and OSX github actions as well.

Reply:
@jsiirola Thanks for the help here.